lib/c-hyper.c

```diff
@@ -464,7 +464,7 @@ CURLcode Curl_hyper_header(struct Curl_easy *data, hyper_headers *headers,
     else
       linelen = 2; /* CRLF ending */
     linelen += (p - n);
-    if(!n)
+    if(!p)
```

Comment:
AFAICS neither n nor p can be null here so I don't know why that check is there. Do you want to adjust your PR to remove it?

Reply:
You mean remove the entire if-statement?

```C
467    if(!p)
468      return CURLE_BAD_FUNCTION_ARGUMENT;
```